docs/content/operating.html.md

```diff
@@ -0,0 +1,233 @@
+logged by all app processes and can also follow the stream as new log lines are
+emitted.
+
+The `flynn-host log $JOB_ID` command on a server will retrieve the logs for
+a specific job and can also follow the stream for that job. A list of all jobs,
+including those that are no longer running can be retrieved with the `flynn-host
+ps -a` command.
+
+Flynn stores logs from jobs in `/var/log/flynn` on each host. There is one log
+file for jobs from each app that has been run on the host. All logs from jobs
+tagged with the same app ID will go into a file named
+`/var/log/flynn/$APP_ID.log`. When a log file reaches 100MB in size, it is
+rotated and a new file is created. One previous rotated log is kept, for a total
+of a maximum 200MB of logs per app per host.
+
+Upstart manages the `flynn-host` daemon and stores the log at
+`/var/log/upstart/flynn-host.log`.
+
+The `flynn-host collect-debug-info` command will collect information about the
+system it is run on along with recent logs from all apps and the `flynn-host`
+daemon. By default it uploads these logs to [Github's
+Gist](https://gist.github.com) service, but they can also be saved to a local
+tarball with the `--tarball` flag.
+
+### Internal Databases
+
+The `controller`, `router`, and `blobstore` components store data in
+a PostgreSQL cluster managed by Flynn. Their databases may be accessed by
+running `flynn -a $APP_NAME pg psql`.
+
+## Updating
+
+There are two ways to update Flynn: in-place and backup/restore. The in-place
+updater is new and we do not consider it to be stable. The most reliable
+update method is backup/restore.
+
+### Backup/Restore
+
+The backup/restore update method involves taking a full backup of the cluster
+and restoring it to a new cluster with the new version of Flynn.
+
+1. Take a backup of the cluster with `flynn cluster backup --file backup.tar`.
+2. Install the new version of Flynn on a new cluster by following [the manual
+   installation instructions](/docs/installation/manual) up to but not including
+   the bootstrap step.
+3. Run the `flynn-host bootstrap` command from the installation guide with an
+   added flag pointing at the cluster backup file: `flynn-host bootstrap
+   --from-backup backup.tar`
+4. Update the DNS records that point at the old cluster to point at the new
+   cluster.
+
+### In-place update
+
+The in-place updater is new and could cause cluster failure. We recommend taking
+a full backup of the cluster first with `flynn cluster backup`.  There is almost
+zero downtime during the cluster update, however database clusters may be
+unavailable for a few seconds while they are updated.
+
+To perform an in-place update of the entire cluster, run `flynn-host update`.
+
+## Adding Hosts
+
+Hosts may be added to an existing cluster by running `flynn-host init` with the
+discovery token or list of host IPs that was used to start the cluster. Care
+should be taken to ensure that the same version of Flynn is installed on all
+hosts.
```

Comment:
We should add a note on how to determine the version of an existing cluster, and then how to install that same version on a new host (setting `FLYNN_VERSION=xxx` when running the install script).